cmd/influx/config.go

```diff
@@ -123,10 +133,8 @@ func (b *cmdConfigBuilder) cmdCreateRunEFn(*cobra.Command, []string) error {
 	}
 
 	pp[b.name] = p
-	if p.Active {
-		if err := pp.Switch(b.name); err != nil {
-			return err
-		}
+	if err := pp.Switch(b.name); err != nil {
```

Comment:
is `Switch` the same as setting the config to active? If so, maybe renaming this to `SetActive` to make that more obvious would help